packages/cli/package.json

```diff
@@ -48,7 +48,9 @@
     "@vercel/node": "1.12.2-canary.4",
     "@vercel/python": "2.0.6-canary.4",
     "@vercel/ruby": "1.2.8-canary.3",
-    "update-notifier": "4.1.0"
+    "update-notifier": "4.1.0",
+    "vercel-plugin-middleware": "0.0.0-canary.5",
```

Comment:
These were added back to dependencies from devDependencies as they're never `import`ed directly and so were not getting bundled.

Also, the loading plugins currently matches against a project's package.json for dependencies/devDependencies and the CLI's package.json for dependencies.